css/style.css

```
@@ -0,0 +1,153 @@
+/*=============================================================================
+ベース
+==============================================================================*/
+*, *::before, *::after {
+  box-sizing:inherit;
+}
+
+html {
+  box-sizing:border-box;
+}
+
+body {
+  color:#333;
+  font-family:'Roboto','sans-serif';
+  line-height:1.8;
+  font-size:14px;
+}
+
+img {
+  width:100%;
+  height:auto;
+  vertical-align:bottom;
+}
+
+a {
+  text-decoration:none;
+  color:inherit;
+}
+
+a:hover {
+  opacity:0.7;
+}
+/*=============================================================================
+グローバル
+==============================================================================*/
+.btn {
+  display:flex;
+  justify-content:center;
+  align-items:center;
+  background-color:#ffa800;
+  color:#fff;
+  border-radius:9999px;
+  border-style:none;
+  cursor:pointer;
+  font-weight:bold;
+}
+
+.pcNone {
+  display:none;
+}
+
+/*============================================================================*/
+header {
+  background-color:#fff;
+  position:sticky;
+  top:0;
+  box-shadow:rgba(0, 0, 0, 0.3) 1px 1px 3px;
+}
+
+header .inner {
+  max-width:980px;
+  width:96%;
+  height:60px;
+  display:flex;
+  justify-content:space-between;
+  align-items:center;
+  margin:auto;
+}
+
+header .logo a img {
+  display:flex;
+  width:100px;
+  height:20px;
+}
+
+header .nav .list {
+  display:flex;
+  align-items:center;
+}
+
+header .nav .list .item + .item {
+  margin-left:20px;
+}
+
+header .btn {
+  width:132px;
+  height:30px;
+}
+
+header .fas {
+  display:none;
+}
+
+.firstView {
+  background-image:url(../img/first-view.jpg);
+  background-size:cover;
+  background-position:center;
+  color:#fff;
+  padding-top:60px;
+  height:600px;
+}
+
+.firstView .inner {
+  max-width:662px;
+  width:96%;
+  margin:auto;
+}
+
+.firstView .text {
+  margin-top:40px;
+  text-align:center;
+}
+
+.firstView .conditions {
+  border-top:1px solid #fff;
+  margin:40px auto;
```

marginの方向が統一されてないです。
```suggestion
  margin:40px auto 0;
```
にして、`.btnWrap`で`margin-top; 40px;`を当てましょう。

---

> marginの方向が統一されてないです。
> 
> にして、`.btnWrap`で`margin-top; 40px;`を当てましょう。

修正しました。
59b9eaaf365c73b1050e7acf6f16a3c89e081f45